qemu/tests/numa_memdev_options.py

```diff
@@ -88,7 +88,11 @@ def check_memory_in_procfs(test, params, vm):
         if mem_path and (mem_path not in numa_maps):
             test.fail("memdev = %s: mem-path '%s' is not in numa_maps '%s'!"
                       % (mem_dev, mem_path, numa_maps))
-        policy_numa = numa_maps.split()[1].split(':')
+        if re.search("many", numa_maps):
+            policy_numa = numa_maps.split()[1].split()
+            policy_numa.append(numa_maps.split()[2].split(':')[1])
+        else:
+            policy_numa = numa_maps.split()[1].split(':')
```

Comment:
```suggestion
numa_maps = re.sub('\s+\(many\)', '',numa_maps)
policy_numa = numa_maps.split()[1].split(':')
```

Reply:
@mcasquer   Let's use the `r` mode:  ` numa_maps = re.sub(r'\s+\(many\)', '',numa_maps)`.

Reply:
> @mcasquer Let's use the `r` mode: ` numa_maps = re.sub(r'\s+\(many\)', '',numa_maps)`.

@yanan-fu now CI checks passed, thanks!